phd-tests/framework/src/artifacts.rs

```diff
@@ -394,8 +381,6 @@ mod test {
     #[test]
     fn verify_raw_toml() {
         let raw = r#"
-            remote_root = "https://10.0.0.255"
-
             [guest_images.alpine]
             guest_os_kind = "alpine"
             metadata.relative_local_path = "alpine.raw"
```

Comment:
Can't comment directly (thanks Github) but looks like this test is still using `relative_remote_path`?